### PR TITLE
Garbage index fix

### PR DESCRIPTION
**Pull request by Cyl07**

### CODE REVIEW

File: src/Controller/ProfileController.php [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+#[Route('/profile', name: 'profile_')]
+class ProfileController extends AbstractController
+{
+    #[Route('/{id}', name: 'show')]
+    public function show(User $user): Response
+    {
+        return $this->render('profile/show.html.twig', [
+            'user' => $user,
+        ]);
+    }
```
Comment (rocambille) [inline src/Controller/ProfileController.php]: should be:

    #[Route('/profile', name: 'profile')]
    public function show(): Response
    {
        return $this->render('profile/show.html.twig', [
            'user' => $this->getUser(),
        ]);
    }

no?

**Review by rocambille — CHANGES_REQUESTED** [review]

**Comment by rocambille** [comment]
conflicts to resolve

File: src/Controller/ProfileController.php (hunk 1 above)
Comment (Cyl07) [reply src/Controller/ProfileController.php]: This feature is for future application where you want to see the profile of another person of your community.

Comment (rocambille) [reply src/Controller/ProfileController.php]: that's the purpose of UserController::show, isn't it?

Comment (Cyl07) [reply src/Controller/ProfileController.php]: We want to separate user informations (which include password and other things) and profile which describe the page seen by the user.

**Comment by rocambille** [comment]
https://github.com/WildCodeSchool/reims-202109-php-project3-incit-easy/runs/4549718652?check_suite_focus=true

**Pull request merged**